edenai_apis/apis/perplexityai/perplexityai_api.py

```
@@ -121,3 +134,165 @@ def text__chat(
                         stream=self.__text_to_json(lst_data)
                     ),
                 )
+
+    def llm__chat(
+        self,
+        messages: List = [],
```

Ruff (0.13.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)